.github/workflows/pr-ci.yml

```diff
@@ -23,8 +23,11 @@ jobs:
     steps:
       - name: Install dependencies (Ubuntu)
         if: runner.os == 'Linux'
-        run: sudo apt-get update && sudo apt-get install -y clang doxygen gcc gcc-10 gcc-9 libstdc++-10-dev libstdc++-9-dev ninja-build python3-pip python3-setuptools valgrind
```

Comment:
I wonder if this needs to add `g++-10 g++-9`

Reply:
Maybe not, as it passes tests without. Although I'm actually wondering why apt is installing g++ without a request for it - g++ might be in the 'Suggested:' section, which is then config dependent.

Reply:
I wondered that.  It may be that it's already installed in the actions host as well.  Looking at the range of compilers is what I was going to look at next.